.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,17 @@
-<!-- ## Description -->
+<!-- Please read https://github.com/tripleblindmarket/covid-safe-paths/wiki/Pull-Request-Best-Practices for recommended best practices before opening your first pull request -->
+
+# Description:
 
 <!-- Description of what the PR does -->
 
-#### Linked issues:
+## Linked issues:
```

Comment:
These headings will be massive!! Can you restore h4?

Reply:
sure thing :)